js/repeater.js

```diff
@@ -76,7 +76,8 @@
 		this.$pageSize.selectlist();
 		this.$primaryPaging.find('.combobox').combobox();
 		this.$search.search({
-			searchOnKeyPress: this.options.searchOnKeyPress
+			searchOnKeyPress: this.options.searchOnKeyPress,
+			allowCancel: true
```

Comment:
The tech specs didn't cover whether it should always be cancelable, so allowing cancels seemed the less dangerous assumption. I can definitely allow passthrough for Repeater if you'd like it to.